datasets/us/hi/med_exclusions/us_hi_med_exclusions.yaml

```diff
@@ -0,0 +1,43 @@
+title: US Hawaii State Medicaid Exclusions
+entry_point: crawler.py
+prefix: us-medhi
+coverage:
+  frequency: daily
+  start: "2024-10-08"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Medical providers excluded from Hawaii's Medicaid program.
+description: |
+  The State of Hawai'i Department of Human Services maintains a list of providers
+  who have been excluded from the state program.
```

Comment:
Please add

> Excluded individuals and entities are listed according to the type of provider they were at the time of exclusion; however, they are excluded from participating in providing services in the Medicaid program in all categories of service and in any capacity.

with a blank line in between the existing and the new paragraphs so that we get separation of the paragraphs and quotes on the web page.